lib/evmmax/evmmax.cpp

```diff
@@ -0,0 +1,113 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <evmmax/evmmax.hpp>
+
+using namespace intx;
```

Comment:
Doesn't this conflict with the `uint` definition we have in `evmmax`?

Reply:
No. In evmmax.hpp we have `ModArith::uint`.